### PR TITLE
I os api 0120

### PR DESCRIPTION
**Pull request by SuQiang2007**
Added 4 APIs, they are:
1. authentication->GuestLogin;
2. marketplace->CheckStatusOfMinting;
3. marketplace->UpdateNFTProperties;
4. marketplace->CheckStatusOfTransactions;

### CODE REVIEW

**Review by SuQiang2007 — COMMENTED** [review]
Finish self reviewing.

File: data/api-reference/200-ios/100-authentication/100-StartLogin.mdx [hunk 1]
```diff
@@ -3,14 +3,12 @@ title: StartLogin
 ---
 
 # `StartLogin`
+Calling this api would popup a dialog, user can finish login flow on it. In which dialog, user can login with third method like google, twitter. Or he can login with his email which registered on our website.
 
-Calling this API would popup a dialog, user can finish login flow on it. In which the user can login with another method like Google or Twitter. Or they can login with their registered email.
-
-```swift
+```Swift
```
Comment (codebender828) [inline data/api-reference/200-ios/100-authentication/100-StartLogin.mdx:8]: please leave this as `swift`

File: data/api-reference/200-ios/200-marketplace/201-CheckStatusOfMinting.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: CheckStatusOfMinting
+---
+
+# `CheckStatusOfMinting`
+Get status of a minting operation. Because minting may has some delay before success.
+```Swift
```
Comment (codebender828) [inline data/api-reference/200-ios/200-marketplace/201-CheckStatusOfMinting.mdx:7]:
```suggestion
```swift
```

File: data/api-reference/200-ios/200-marketplace/202-UpdateNFTProperties.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: UpdateNFTProperties
+---
+
+# `UpdateNFTProperties`
+Update properties of a NFT.
+```Swift
```
Comment (codebender828) [inline data/api-reference/200-ios/200-marketplace/202-UpdateNFTProperties.mdx:7]:
```suggestion
```swift
```

File: data/api-reference/200-ios/200-marketplace/203-CheckStatusOfTransactions [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: CheckStatusOfTransactions
+---
+
+# `CheckStatusOfTransactions`
+
+Get status of transactions by their signatures.
+```Swift
```
Comment (codebender828) [inline data/api-reference/200-ios/200-marketplace/203-CheckStatusOfTransactions:8]:
```suggestion
```swift
```

**Review by codebender828 — CHANGES_REQUESTED** [review]
Hi. thanks for the pr @SuQiang2007 

Please change the code snippet languages to lower case. 
It should be: `swift` not `Swift`

File: data/api-reference/200-ios/100-authentication/100-StartLogin.mdx (hunk 1 above)
Comment (SuQiang2007) [reply data/api-reference/200-ios/100-authentication/100-StartLogin.mdx:8]: Ok,will mention that.

**Review by SuQiang2007 — COMMENTED** [review]
Checked

**Pull request merged**